package.json

```diff
@@ -11,6 +11,14 @@
     "node": ">=5"
   },
   "scripts": {
+    "install-all": "npm run install-cli && npm run install-extension && npm run install-viewer",
+    "install-cli": "npm --prefix ./lighthouse-cli install ./lighthouse-cli",
+    "install-extension": "npm --prefix ./lighthouse-extension install ./lighthouse-extension",
+    "install-viewer": "npm --prefix ./lighthouse-viewer install ./lighthouse-viewer",
```

Comment:
this and the other `installs` dont need to specify the path, afaict.

can be just `npm install -C ../lighthouse-viewer`